tt_eager/tt_dnn/op_library/transformer_tms/kernels/dataflow/reader_ssm_prefix_scan.cpp

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "dataflow_api.h"
+
+void kernel_main() {
+    uint32_t num_tiles_per_core = get_arg_val<uint32_t>(0);
+    constexpr uint32_t cb_a_in = get_compile_time_arg_val(0);
+    constexpr uint32_t cb_bx_in = get_compile_time_arg_val(1);
+
+    cb_push_back(cb_a_in, num_tiles_per_core);
+    cb_push_back(cb_bx_in, num_tiles_per_core);
+}
```

Comment:
What's the point of this kernel? If input is sharded and nothing needs to be done, do we even need this synchronization.

Reply:
I understood that consumers of `cb_a_in` and `cb_bx_in` in the compute kernel would sit on their `cb_wait_front` calls until `cb_push_back` was called here - maybe that is wrong?

Reply:
If there is literally nothing happening, you can just use the CBs directly in the compute

Reply:
Cool, I'll just remove the reader then.

Reply:
Actually, instead of moving the push calls to the compute, it might be better to leave them in the reader since the next version of this kernel will require some additional logic to read in the previous hidden states. Is that OK?

Reply:
Yup makes sense if it's a placeholder